task1.py

```diff
@@ -0,0 +1,5 @@
+print('Multi-paradigm movie programming','Task1')
+print('Koshma Vladyslav','IKM-221a')
+str_name = ('Vladyslav' for _ in range(45))
+print(','.join(str_name))
+print((195 + 1,3/8))/(18,2 - (3,8 * 3,8) + 19,3)
```

Comment:
take care about spaces around the operators
```suggestion
print((195 + 1,3 / 8)) / (18,2 - (3,8 * 3,8) + 19,3)
```